integreat_cms/static/src/css/style.scss

```diff
@@ -561,7 +561,38 @@ datalist option {
 }
 
 /* Special word break for feedback list */
-.feedback-comment-full {
-    @apply break-words whitespace-pre-line;
-    word-break: break-word;
+.feedback-entry {
+    @apply flex flex-nowrap items-center;
+
+    span {
+        @apply overflow-hidden overflow-ellipsis whitespace-nowrap;
+    }
+
+    .toggle-feedback-entry {
+        @apply hover:scale-125 cursor-pointer inline-flex;
+    }
+
+    .less {
+        @apply hidden;
+    }
+
+    .more {
+        @apply inline-flex;
+    }
+
+    &.active {
+        @apply flex-wrap;
+
```

Comment:
```suggestion
```
I think @charludo's original suggestion got lost somewhere along the way - I kind of agree that it looks nicer when the icons stay to the right of the text when expanded, but obviously that's mostly personal preference.